contrib/gorilla/mux/mux.go

```diff
@@ -7,6 +7,7 @@
 package mux // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux"
 
 import (
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
```

Comment:
We sort import in a specific order, please put this alongside other gopkg imports.